apps/console/src/features/email-templates/components/email-template-editor.tsx

```diff
@@ -82,10 +82,16 @@ export const EmailTemplateEditor: FunctionComponent<EmailTemplateEditorPropsInte
      * Write content to iFrame.
      */
     const writeToIframe = (): void => {
-        const iframeDoc = iframe?.current?.contentDocument || iframe?.current?.contentWindow?.document;
-        if (iframeDoc) {
-            iframeDoc.body.innerHTML = content;
-        }
+        // This interval will wait till the iframe is initialized to show content.
+        const inter = window.setInterval(function() {
+            const iframeDoc = iframe?.current?.contentDocument || iframe?.current?.contentWindow?.document;
+
+            if(iframeDoc.readyState == "complete") {
+
+                window.clearInterval(inter);
```

Comment:
E.g. Use complete word when declaring variables, unless its a commonly using term.

```suggestion
                window.clearInterval(interval);
```

Reply:
Fixed with b25631a